docs/content/en/docs/tasks/troubleshoot/_troubleshooting.md

```diff
@@ -232,6 +232,12 @@ If this IP is not present on any control plane VM, make sure the `network` has a
 
 {{% content "../../reference/vsphere/domains.md" %}}
 
+If the IPv4 IPs are assigned to the VM and you have the workload kubeconfig under `<cluster-name>/<cluster-name>-eks-a-cluster.kubeconfig`, you can use it to check `vsphere-cloud-controller-manager` logs.
```

Comment:
Can we call out the specific error message that could indicate a misconfig of the VMs having access to the vcenter api?

Reply:
I agree. Maybe you could show a snippet of output that illustrates what the message might look like.

Reply:
done